.coderabbit.yaml

```
@@ -0,0 +1,23 @@
+language: 'ko-KR'
+early_access: false
+reviews:
+  profile: 'chill'
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: false
+  review_status: true
+  collapse_walkthrough: false
+reviews:
```

YAMLlint (1.35.1)

[error] duplication of key "reviews" in mapping

(key-duplicates)